types/gulp/index.d.ts

```diff
@@ -14,6 +14,7 @@ declare type Strings = string|string[];
 
 declare namespace gulp {
     interface Gulp extends Orchestrator {
+        task(name: string): never;
```

Comment:
I see.  Gulp doesn't extend Orchestrator's task in a straightforward way...  It's very interesting there is such kind of workaround.  Thanks!